packages/semver/src/schematics/ng-add/schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "ngAdd",
+  "title": "@jscutlery/semver ng-add schematic",
+  "type": "object",
+  "properties": {
+    "syncVersions": {
+      "description": "Lock/sync versions between packages",
```

Comment:
Should we use a default value or make it required? @yjaaidi

Reply:
I think that the default behavior should be independent and then let users opt-in using `sync-versions`.
Let's use a default of false.